code-studio/src/js/components/Attachments.jsx

```diff
@@ -0,0 +1,73 @@
+/* global dashboard */
+
+var assetsApi = require('@cdo/apps/clientApi').assets;
+var assetListStore = require('../assets/assetListStore');
+
+var styles = {
+  button: {
+    float: 'left',
+    marginRight: '5px'
+  },
+  attachment: {
+    background: '#08c',
+    color: '#fff',
+    borderRadius: '4px',
+    padding: '0 5px',
+    margin: '4px 4px 4px 0',
+    float: 'left',
+    fontSize: '0.8em'
+  }
+};
+
+/**
+ * An attachment list component.
+ */
+var Attachments = React.createClass({
+  propTypes: {},
+
+  getInitialState: function () {
+    return {loaded: false};
+  },
+
+  componentWillMount: function () {
+    assetsApi.ajax('GET', '', this.onAssetListReceived);
+  },
+
+  onAssetListReceived: function (xhr) {
+    assetListStore.reset(JSON.parse(xhr.responseText));
+    if (this.isMounted()) {
+      this.setState({loaded: true});
```

Comment:
This is messy because our simple store doesn't provide events for when things change.  Maybe now is a good time to upgrade `assetListStore` to Redux 😄 .